File: JanperCab/3-Application/Dtos/DuraformOrder/DuraformOrderDto.cs
using _1_Domain;
using _3_Application.Dtos.DuraformForm;

namespace _3_Application.Dtos.DuraformOrder
{
    public class DuraformOrderDto : DuraformFormDto
    {
        public int OrderNumber { get; set; }

        public OrderStatus OrderStatus { get; set; }
    }
}

File: JanperCab/5-JanperCab/ClientApp/src/app/_models/duraform-order/DuraformOrderDto.ts
import { OrderStatus } from './../../_enums/OrderStatus';
import { DuraformFormDto } from './DuraformFormDto';
import { DuraformOrderTypeKey } from 'src/app/_enums/DuraformOrderTypeKey';

export class DuraformOrderDto extends DuraformFormDto {
  orderNumber: number;
  orderStatus: OrderStatus;

  constructor() {
    super();
    this.orderType = DuraformOrderTypeKey.Order;
  }
}
